Products/ZenModel/tests/testFileSystem.py

```diff
@@ -29,11 +29,11 @@ def afterSetUp(self):
 
 
     def testSetManageIp(self):
-        self.fs.setManageIp('1.2.3.4/24')
-        self.assert_(self.fs.getManageIp() == '1.2.3.4/24')
-        self.assert_(self.dev.getManageIp() == '1.2.3.4/24')
-        self.dev.setManageIp('2.3.4.5/24')
-        self.assert_(self.fs.getManageIp() == '2.3.4.5/24')
```

Comment:
`setManageIp` needs to be able to accept `/24` IP notation.  This is likely to break existing code that expects `getManageIp` to accept such IP addresses.